input/ImplementationGuide-saraalert-fhir.json

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
We don't have to check this file into source control, since SUSHI will generate it to `fsh-generated/resources`. So we can remove this file, and then we just have to change the `ig` line in `ig.ini` to be:
```
ig = fsh-generated/resources/ImplementationGuide-saraalert-fhir.json
```

Reply:
https://github.com/SaraAlert/saraalert-fhir-ig/pull/6/commits/e90a101d4dc0c7a2e8219aacd6e9bf7cf6da4d62